acceptance/framework/environment/environment.go

```diff
@@ -187,6 +191,7 @@ func (k kubernetesContext) ControllerRuntimeClient(t testutil.TestingTB) client.
 
 	client, err := client.New(config, client.Options{Scheme: s})
 	require.NoError(t, err)
+	logf.SetLogger(zap.New(zap.WriteTo(os.Stdout), zap.UseDevMode(true)))
```

Comment:
need to rework this to not use zap, we're hitting the same issue listed here from the controller-runtime upgrade https://github.com/solo-io/gloo/issues/8548